pkg/kubelet/cm/cpumanager/cpu_assignment.go

```diff
@@ -453,7 +453,7 @@ func takeByTopologyNUMAPacked(topo *topology.CPUTopology, availableCPUs cpuset.C
 		return acc.result, nil
 	}
 	if acc.isFailed() {
-		return cpuset.New(), fmt.Errorf("not enough cpus available to satisfy request")
+		return cpuset.New(), fmt.Errorf("not enough cpus available to satisfy request: %d requested, only %d available", numCPUs, availableCPUs.Size())
```

Comment:
I'd go for a terser form: "requested %d, available %d" or even "requested=%d, available=%d". The current wording yeilds a nicer sentence, but IMO here practicality beats nice. Applies to everything below.

Reply:
Done in [d4a5a08](https://github.com/kubernetes/kubernetes/pull/121059/commits/d4a5a085a84b3d2c3853b589787e25faec33e380)